diff --git a/coldfront/config/urls.py b/coldfront/config/urls.py
index 78d9589..f1cd16a 100644
--- a/coldfront/config/urls.py
+++ b/coldfront/config/urls.py
@@ -52,6 +52,9 @@ if "mozilla_django_oidc" in settings.INSTALLED_APPS:
 if "django_su.backends.SuBackend" in settings.AUTHENTICATION_BACKENDS:
     urlpatterns.append(path("su/", include("django_su.urls")))

+if 'coldfront_plugin_api' in settings.INSTALLED_APPS:
+    urlpatterns.append(path('api/', include('coldfront_plugin_api.urls')))
+

 def export_as_json(modeladmin, request, queryset):
     response = HttpResponse(content_type="application/json")
